mix.exs

```diff
@@ -38,7 +38,7 @@ defmodule Membrane.Opus.Plugin.Mixfile do
       {:bunch, "~> 1.3"},
       {:membrane_core, "~> 0.9.0"},
       {:membrane_opus_format, "~> 0.3.0"},
-      {:membrane_caps_audio_raw, "~> 0.6.0"},
```

Comment:
You may do the usual housekeeping before the release - 
* `Apache-2.0` (with dash) license
* `formatters: ["html"]` for docs